src/unpack.js

```diff
@@ -64,6 +64,23 @@ function unpackRelease(unpackFolder, targetFolderName, archiveFile, progressElem
 
                 unpacker = path.join(remote.app.getAppPath(), "tools/7zip/7za");
             }
+        } else if(os.platform() === "darwin"){
+            // Find from PATH
+            unpacker = which.sync("7za", {nothrow: true});
+
+            if(!unpacker){
+                // Use packed in version //
+                console.log("No 7za found in PATH, using packed in one");
+
+                unpacker = path.join(remote.app.getAppPath(), "tools/7zip/7za");
```

Comment:
Does this actually work? Isn't this the linux executable?

Reply:
Actually no, let me remove that.